scripts/zones/Aydeewa_Subterrane/npcs/blank_omens.lua

```diff
@@ -11,7 +11,12 @@ function onTrade(player,npc,trade)
 end
 
 function onTrigger(player,npc)
-    if player:getCharVar("OmensProgress") == 4 then
+local omensProgress = player:getCharVar("OmensProgress")
```

Comment:
add indentation here.  otherwise, looks good!  thanks for the fix.

Reply:
oh darn, i was hella tired i didnt notice. fixed =)